emcc.txt is read from emcc.py for --help. Move it to more reasonable location.
--- a/emcc.py
+++ b/emcc.py
@@ -153,7 +153,7 @@
 
 emcc: supported targets: llvm bitcode, javascript, NOT elf
 (autoconf likes to see elf above to enable shared object support)
-''' % (open(shared.path_from_root('site', 'build', 'text', 'docs', 'tools_reference', 'emcc.txt')).read())
+''' % (open(shared.path_from_root('emcc.txt')).read())
     exit(0)
 
   elif sys.argv[1] == '--version':
